Componentss/ratingSlider.html

```diff
@@ -0,0 +1,167 @@
+                    Github
+                </a>
+            </li>
+            <li>
+                <button class="menu">
+                    <i class="bi bi-list"></i>
+                </button>
+            </li>
+        </ul>
+    </div>
+
+    <nav class="sidebar">
+        <!-- Getting Started Navigation Start -->
+        <div>
+            <div class="list-topic">
+                <a href="#">GETTING STARTED</a>
+            </div>
+            <ul class="list-subtopic">
+                <li>
+                    <a href="./installation.html">Installation</a>
+                </li>
+                <li>
+                    <a href="./typography.html">Typography</a>
+                </li>
+            </ul>
+        </div>
+        <!-- Getting Started Navigation End -->
+
+        <!-- Components Navigation Start -->
+        <div>
+            <div class="list-topic">
+                <a class="" href="#">COMPONENTS</a>
+            </div>
+            <ul class="list-subtopic">
+                <li>
+                    <a href="./avatar.html">Avatar</a>
+                </li>
+                <li>
+                    <a href="./alert.html">Alert</a>
+                </li>
+                <li>
+                    <a href="./badge.html">Badge</a>
+                </li>
+                <li>
+                    <a href="./button.html">Button</a>
+                </li>
+                <li>
+                    <a href="./cards.html">Card</a>
+                </li>
+                <li>
+                    <a href="./image.html">Image</a>
+                </li>
+                <li>
+                    <a href="./input.html">Input</a>
+                </li>
+                <li>
+                    <a href="./list.html">List</a>
+                </li>
+                <li>
+                    <a href="./text_utilities.html">Text Utilities</a>
+                </li>
+                <li>
+                    <a href="./snackbar.html">Snackbar</a>
+                </li>
+                <li>
+                    <a href="#navbar">Navbar</a>
+                </li>
+                <li>
+                    <a class='active' href="./ratingSlider.html">Slider</a>
+                </li>
+                <li>
+                    <a href="#rating">Rating</a>
+                </li>
+            </ul>
+        </div>
+        <!-- Components Navigation End -->
+    </nav>
+    <!-- Sidebar Section End -->
+
+    <!------------------------------------------- Main Section ------------------------------------------>
+
+    <section class="main-container main">
+        <div class="pd-1-lr pd-1-bot">
+
+            <!-- Slider -->
+
+            <h1 class="mg-point8-tb title sub-head hover-underline-animation">Slider</h1>
+            <p>An slider is a graphical representation of a user or the user's character or persona. Lorem ipsum dolor
+                sit, amet consectetur adipisicing elit. Laboriosam, repellendus?
+
```

Comment:
extra empty line  inside the < p > tag maybe you were trying to give a empty line  after the < p > tag

Reply:
It's by mistake. Will remove that.
Thankyou!!